### PR TITLE
Improved the UI of ebpf and other svg's for better visibility in dark mode

### LINKED ISSUE #5070: Dark mode version of eBPF logo on Meshmap Page
### Description
Currently, in the dark mode the "ebpf logo" is not visible accurately on the [catalog page](https://layer5.io/cloud-native-management/meshmap)

### Expected Behavior
A dark-mode friendly ebpf logo should be updated with the current one

### Screenshots
Screenshot 2023-10-21 233015

### PR DESCRIPTION
**Pull request by Suswetha6**
**Description**

This PR fixes #5070 

**Notes for Reviewers**
I've added a new property in the theme objects in themeStyles.js and updated the css of svg in meshmap section.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit a486635f3ef731735081191dc4c0b1d05a03ddfe at: https://66bda7570d0bf57b1954d8ab--layer5.netlify.app

**Comment by narasimha-1511** [comment]
Hey @Suswetha6 please sign the commits using -s , now you need to rebase and squash them

**Comment by narasimha-1511** [comment]
Screenshot 2024-08-16 at 2 26 54 AM
You should not add the boder box or something kindly change the svg itself which appears better in the dark mode

File: src/sections/Meshmap/meshmap-catalog.js [hunk 1]
```diff
@@ -120,6 +120,8 @@ box-shadow: 0px 6px 5px 0px rgb(0 0 0 / 25%);
     max-width:5.6rem;
     display:block;
     margin: 0 auto;
+    border: 2px solid ${props => props.theme.borderColorWhiteToTransparent};
```
Comment (narasimha-1511) [inline src/sections/Meshmap/meshmap-catalog.js:123]: remove these borders

**Review by narasimha-1511 — CHANGES_REQUESTED** [review]
kindly remove these borders and update the svg

File: CNAME
Comment (Ashparshp) [inline CNAME:1]: @Suswetha6, why did you remove this file?

**Comment by Ashparshp** [comment]
@narasimha-1511, do we have a svg for this? and @Suswetha6, please fix the DCO sign-off. If you're unaware, read more about it in the contributing guide.

**Comment by Ashparshp** [comment]
@Suswetha6, Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by Ashparshp** [comment]
@Suswetha6, please join. The meeting has started.

**Comment by Ashparshp** [comment]
@Suswetha6, any luck?

**Comment by Ashparshp** [comment]
@Suswetha6, checking again?

**Comment by Suswetha6** [comment]
Hey @narasimha-1511  , sorry i was not available. I am not sure how to do the DCO sign-off, i will read about it and do it by eod

**Comment by l5io** [comment]
🚀 Preview for commit 0aada74a41115d3bad8b8e09a0738bdb4ea6ed14 at: https://66c6e5c9cde1b814330e07c2--layer5.netlify.app

**Comment by Ashparshp** [comment]
@Suswetha6, any update?

File: CNAME
Comment (vishalvivekm) [inline CNAME:1]: @Suswetha6 Why is this file deleted ? will you revert this change please ?

**Review by vishalvivekm — CHANGES_REQUESTED** [review]
I don't see any difference this PR makes to the svgs what we have right now: 
image

File: src/theme/app/themeStyles.js
Comment (vishalvivekm) [inline src/theme/app/themeStyles.js:1]: will you please sound off why are these theme changes are being introduced ? @Suswetha6

**Pull request closed without merging**